dataactbroker/README.md

```diff
@@ -616,14 +618,16 @@ Example output:
           "error_name": "rule_failed",
           "field_name": "award_financial",
           "occurrences": "11",
-          "rule_failed": "fain, uri"
+          "rule_failed": "fain, uri",
+          "original_label":""
```

Comment:
That particular rule does not have a label in the current rule definition, as it originally came in through a JIRA ticket and not through the practices and procedures document.  That said, C14 is now present and is similar but not quite the same rule since it doesn't include piid.  That said, since we've fixed the bug on cross-file rule_failed, I'll fix the text on that.